plugins/ship_it.py

```diff
@@ -8,13 +8,19 @@ class Ship_it(BotPlugin):
     """
     Show motivational ship it squirrel images.
     """
-
     # start ignoring LineLengthBear PyCodeStyleBear
     IMAGES = [
         'http://i.imgur.com/DPVM1.png',
         'http://d2f8dzk2mhcqts.cloudfront.net/0772_PEW_Roundup/09_Squirrel.jpg',
         'http://www.cybersalt.org/images/funnypictures/s/supersquirrel.jpg',
         'http://www.zmescience.com/wp-content/uploads/2010/09/squirrel.jpg',
+        'https://memegenerator.net/img/instances/500x/16945547/jenkins-working-ship-it.jpg',
+        'https://memegenerator.net/img/instances/500x/54432217/executive-code-ship-it.jpg',
+        'https://lasfloressquirrels.files.wordpress.com/2010/05/squirrel-pirate.jpg',
```

Comment:
I see a lot many http 500 error
you can see this link https://www.lifewire.com/500-internal-server-error-explained-2622938

idk maybe this could help you fixing it

Reply:
I think these 404 or 500 error can be fixed if you remove https and add http instead of that

maybe the encrypted connect is making a problem

Its just a trick I am not sure it will work